JobQuiver/JobQuiver
# Saved results route

**WHAT DOES THIS PR DO:**
1. Adds general structure for the /savedResults routes. 
2. Updates scripts for better development environment. 

**HOW TO TEST THIS PR:**
1. Run `gh pr checkout 4 && npm install && npm run build`. 
2. Run the various scripts—`npm start`, `npm run dev`, `npm run client`, `npm run server`—and verify that they all work. You should be able to access the FE (lh8080) and make requests to the BE (lh3000). 
3. While in the dev environment, verify that you can make changes to the FE and the BE that will be hot reloaded. 
4. In whatever environment you wish, verify that you can make requests to the /savedResults routes and get the appropriate messages back. (I tested in postman the browser). 
5. Verify that the commented out middleware logic generally looks good. (It will need some tweaking once we actually have our data, but that is to be expected.)

**ADDITIONAL NOTES:** 
1. @michaelweberjr we're going to hit some merge conflicts with https://github.com/JobQuiver/JobQuiver/pull/3 depending on the merge order.
